quickwit/quickwit-search/src/collector.rs

```diff
@@ -308,6 +312,33 @@ impl QuickwitSegmentCollector {
         let (sort_value, sort_value2) =
             self.score_extractor.extract_typed_sort_value(doc_id, score);
 
+        if let Some(search_after) = &self.search_after {
+            let search_after_value1 = search_after.sort_value.and_then(|v| v.sort_value);
+            let search_after_value2 = search_after.sort_value2.and_then(|v| v.sort_value);
+            let orders = &self.top_k_hits.sort_key_mapper;
+            let mut cmp_result = orders
+                .order1
+                .compare_opt(&sort_value, &search_after_value1)
+                .then_with(|| {
+                    orders
+                        .order2
+                        .compare_opt(&sort_value2, &search_after_value2)
+                });
+            if !search_after.split_id.is_empty() {
+                // TODO actually it's not first, it should be what's in _shard_doc then first then
```

Comment:
I don't understand this comment.
Also "_shard_doc" is an elasticsearch concept.

Reply:
yes, but I expect people using search_after on a PIT (which we don't really have yet) to want it for tie-breaking. Note that in ES, you can't use `_shard_doc` outside of a PIT. I take it it's really something related to \*that document\* at \*that time\*, so it's something we can reasonably